packit/distgit.py

```diff
@@ -176,6 +195,7 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
         """
         upload files (archive) to the lookaside cache
         """
+        # TODO: can we check if the tarball is already uploaded so we don't have ot re-upload?
```

Comment:
Hmm, that would be nice. Maybe rebase-helper can help. Or some call to lookaside cache web interface.

Reply:
yup, exactly